# Translation of Odoo Server.
# This file contains the translation of the following modules:
# * base_exception
#
# Translators:
# OCA Transbot <transbot@odoo-community.org>, 2017
msgid ""
msgstr ""
"Project-Id-Version: Odoo Server 10.0\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2017-05-01 10:38+0000\n"
"PO-Revision-Date: 2018-07-22 11:30+0000\n"
"Last-Translator: derKonig <fshahy@gmail.com>\n"
"Language-Team: Persian (https://www.transifex.com/oca/teams/23907/fa/)\n"
"Language: fa\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Plural-Forms: nplurals=1; plural=0;\n"
"X-Generator: Weblate 3.0.1\n"

#. module: base_exception
#: model_terms:ir.ui.view,arch_db:base_exception.view_exception_rule_form
msgid "<code>self</code>: Record on which the rule is evaluated."
msgstr ""

#. module: base_exception
#: model:ir.model.fields,field_description:base_exception.field_exception_rule__active
msgid "Active"
msgstr "فعال"

#. module: base_exception
#: model:ir.model.fields,field_description:base_exception.field_exception_rule__model
msgid "Apply on"
msgstr "اعمال بر"

#. module: base_exception
#: model_terms:ir.ui.view,arch_db:base_exception.view_exception_rule_form
msgid "Archived"
msgstr ""

#. module: base_exception
#: model_terms:ir.ui.view,arch_db:base_exception.view_exception_rule_form
msgid "As well as all the libraries provided in safe_eval."
msgstr ""

#. module: base_exception
#: model:ir.model.fields.selection,name:base_exception.selection__exception_rule__exception_type__by_domain
msgid "By domain"
msgstr ""

#. module: base_exception
#: model:ir.model.fields.selection,name:base_exception.selection__exception_rule__exception_type__by_py_code
msgid "By python code"
msgstr ""

#. module: base_exception
#: model:ir.model.fields,help:base_exception.field_exception_rule__exception_type
msgid ""
"By python code: allow to define any arbitrary check\n"
"By domain: limited to a selection by an odoo domain:\n"
"           performance can be better when exceptions            are "
"evaluated with several records"
msgstr ""

#. module: base_exception
#: model:ir.model.fields,field_description:base_exception.field_exception_rule__create_uid
msgid "Created by"
msgstr "ایجاد شده توسط"

#. module: base_exception
#: model:ir.model.fields,field_description:base_exception.field_exception_rule__create_date
msgid "Created on"
msgstr "ایجاد شده در"

#. module: base_exception
#: model:ir.model.fields,field_description:base_exception.field_exception_rule__description
msgid "Description"
msgstr "توصیف"

#. module: base_exception
#: model:ir.model.fields,field_description:base_exception.field_base_exception__display_name
#: model:ir.model.fields,field_description:base_exception.field_base_exception_method__display_name
#: model:ir.model.fields,field_description:base_exception.field_exception_rule__display_name
#: model:ir.model.fields,field_description:base_exception.field_exception_rule_confirm__display_name
msgid "Display Name"
msgstr "نام نمایشی"

#. module: base_exception
#: model:ir.model.fields,field_description:base_exception.field_exception_rule__domain
msgid "Domain"
msgstr ""

#. module: base_exception
#: model_terms:ir.ui.view,arch_db:base_exception.view_exception_rule_form
msgid ""
"Enter Python code here. Help about Python expression is available in the "
"help tab of this document."
msgstr ""

#. module: base_exception
#: code:addons/base_exception/models/base_exception.py:0
#, python-format
msgid ""
"Error when evaluating the exception.rule rule:\n"
" %s \n"
"(%s)"
msgstr ""

#. module: base_exception
#: model:ir.model,name:base_exception.model_base_exception
#, fuzzy
msgid "Exception"
msgstr "نام استثنا"

#. module: base_exception
#: model:ir.model.fields,field_description:base_exception.field_exception_rule__name
msgid "Exception Name"
msgstr "نام استثنا"

#. module: base_exception
#: model:ir.model,name:base_exception.model_exception_rule
#: model_terms:ir.ui.view,arch_db:base_exception.view_exception_rule_tree
msgid "Exception Rule"
msgstr "قاعده استثنا"

#. module: base_exception
#: model:ir.model,name:base_exception.model_exception_rule_confirm
#, fuzzy
msgid "Exception Rule Confirm Wizard"
msgstr "قاعده استثنا"

#. module: base_exception
#: model:ir.model,name:base_exception.model_base_exception_method
#, fuzzy
msgid "Exception Rule Methods"
msgstr "قاعده استثنا"

#. module: base_exception
#: model_terms:ir.ui.view,arch_db:base_exception.view_exception_rule_form
msgid "Exception Rule Setup"
msgstr ""

#. module: base_exception
#: model:ir.actions.act_window,name:base_exception.action_exception_rule_tree
#: model:ir.ui.menu,name:base_exception.menu_action_exception
msgid "Exception Rules"
msgstr ""

#. module: base_exception
#: model:ir.model.fields,field_description:base_exception.field_exception_rule__exception_type
#, fuzzy
msgid "Exception Type"
msgstr "نام استثنا"

#. module: base_exception
#: model:res.groups,name:base_exception.group_exception_rule_manager
msgid "Exception manager"
msgstr ""

#. module: base_exception
#: model:ir.model.fields,field_description:base_exception.field_base_exception__exception_ids
msgid "Exceptions"
msgstr ""

#. module: base_exception
#: model_terms:ir.ui.view,arch_db:base_exception.view_exception_rule_confirm
msgid "Exceptions Rules"
msgstr ""

#. module: base_exception
#: model:ir.model.fields,field_description:base_exception.field_base_exception__exceptions_summary
#, fuzzy
msgid "Exceptions Summary"
msgstr "نام استثنا"

#. module: base_exception
#: model:ir.model.fields,field_description:base_exception.field_exception_rule_confirm__exception_ids
msgid "Exceptions to resolve"
msgstr ""

#. module: base_exception
#: model:ir.model.fields,help:base_exception.field_exception_rule__sequence
msgid "Gives the sequence order when applying the test"
msgstr ""

#. module: base_exception
#: model_terms:ir.ui.view,arch_db:base_exception.view_exception_rule_form
msgid "Help"
msgstr ""

#. module: base_exception
#: model_terms:ir.ui.view,arch_db:base_exception.view_exception_rule_form
msgid "Help with Python expressions"
msgstr ""

#. module: base_exception
#: model:ir.model.fields,field_description:base_exception.field_base_exception__id
#: model:ir.model.fields,field_description:base_exception.field_base_exception_method__id
#: model:ir.model.fields,field_description:base_exception.field_exception_rule__id
#: model:ir.model.fields,field_description:base_exception.field_exception_rule_confirm__id
msgid "ID"
msgstr "شناسه"

#. module: base_exception
#: model:ir.model.fields,field_description:base_exception.field_base_exception__ignore_exception
#: model:ir.model.fields,field_description:base_exception.field_exception_rule_confirm__ignore
msgid "Ignore Exceptions"
msgstr ""

#. module: base_exception
#: model:ir.model.fields,field_description:base_exception.field_base_exception____last_update
#: model:ir.model.fields,field_description:base_exception.field_base_exception_method____last_update
#: model:ir.model.fields,field_description:base_exception.field_exception_rule____last_update
#: model:ir.model.fields,field_description:base_exception.field_exception_rule_confirm____last_update
msgid "Last Modified on"
msgstr "تاریخ آخرین به‌روزرسانی"

#. module: base_exception
#: model:ir.model.fields,field_description:base_exception.field_exception_rule__write_uid
msgid "Last Updated by"
msgstr "آخرین به روز رسانی توسط"

#. module: base_exception
#: model:ir.model.fields,field_description:base_exception.field_exception_rule__write_date
msgid "Last Updated on"
msgstr "آخرین به روز رسانی در"

#. module: base_exception
#: model:ir.model.fields,field_description:base_exception.field_base_exception__main_exception_id
msgid "Main Exception"
msgstr ""

#. module: base_exception
#: code:addons/base_exception/wizard/base_exception_confirm.py:0
#, python-format
msgid "Only 1 ID accepted, got %r."
msgstr ""

#. module: base_exception
#: model:ir.actions.act_window,name:base_exception.action_exception_rule_confirm
msgid "Outstanding exceptions to manage"
msgstr ""

#. module: base_exception
#: model_terms:ir.ui.view,arch_db:base_exception.view_exception_rule_confirm
msgid "Outstanding exceptions to manager"
msgstr ""

#. module: base_exception
#: model:ir.model.fields,field_description:base_exception.field_exception_rule__code
#: model_terms:ir.ui.view,arch_db:base_exception.view_exception_rule_form
msgid "Python Code"
msgstr ""

#. module: base_exception
#: model:ir.model.fields,help:base_exception.field_exception_rule__code
msgid ""
"Python code executed to check if the exception apply or not. Use failed = "
"True to block the exception"
msgstr ""

#. module: base_exception
#: model:ir.model.fields,field_description:base_exception.field_exception_rule_confirm__related_model_id
#, fuzzy
msgid "Related Model"
msgstr "ایجاد شده در"

#. module: base_exception
#: model:ir.model.fields,field_description:base_exception.field_exception_rule__sequence
msgid "Sequence"
msgstr ""

#. module: base_exception
#: model:ir.model.fields,field_description:base_exception.field_base_exception__smart_search
#: model:ir.model.fields,field_description:base_exception.field_base_exception_method__smart_search
#: model:ir.model.fields,field_description:base_exception.field_exception_rule__smart_search
#: model:ir.model.fields,field_description:base_exception.field_exception_rule_confirm__smart_search
msgid "Smart Search"
msgstr ""

#. module: base_exception
#: code:addons/base_exception/models/base_exception.py:0
#, python-format
msgid ""
"There is a problem of configuration, python code or domain is missing to "
"match the exception type."
msgstr ""

#. module: base_exception
#: model_terms:ir.ui.view,arch_db:base_exception.view_exception_rule_form
msgid "To block the exception use: <code>failed = True</code>"
msgstr ""

#. module: base_exception
#: model_terms:ir.ui.view,arch_db:base_exception.view_exception_rule_form
msgid ""
"Various fields may use Python code or Python expressions. The following "
"variables can be used:"
msgstr ""

#. module: base_exception
#: model_terms:ir.ui.view,arch_db:base_exception.view_exception_rule_confirm
msgid "_Close"
msgstr ""
